concrete/single_pages/dashboard/reports/security.php

```diff
@@ -4,62 +4,84 @@
  */
 use Concrete\Core\Validator\ErrorLevel;
 
-foreach ($result as $validatorName => $resultSet) {
-    $severities = array_map(function(\Concrete\Core\Validator\Site\SiteError $error) {
-        return $error->getSeverity();
-    }, $resultSet);
+if (!count($result)) {
+    ?>
+     <div class="col-sm-12">
+        <div class="alert alert-success text-center">
+            <?= t('No issues found!') ?>
```

Comment:
I prefer having passing checks, but since we aren't defining the requirements we wouldn't actually show anything. Maybe we should be making use of `->getRequirementStrings`?